.circleci/config.yml

```diff
@@ -69,6 +69,31 @@ node-bench-e2e-base: &node-bench-e2e-base
         name: E2E Benchmark
         command: yarn bench:e2e
 
+node-bench-sirun-base: &node-bench-sirun-base
+  resource_class: small
+  working_directory: ~/dd-trace-js
+  steps:
+    - checkout
+    - *yarn-versions
+    - *restore-yarn-cache
+    - *yarn-install
+    - *save-yarn-cache
+    - run:
+        name: Install system deps
+        command: apt-get update && apt-get install -y valgrind wget
+    - run:
+        name: Install sirun
+        command: wget https://github.com/DataDog/sirun/releases/download/v0.1.3/sirun-v0.1.3-x86_64-unknown-linux-gnu.tar.gz && tar zxf sirun-v0.1.3-x86_64-unknown-linux-gnu.tar.gz && mv sirun /usr/bin
+    - run:
+        name: Run sirun
+        command: |
+          cd $SIRUN_TEST_DIR
```

Comment:
Why is this needed?

Reply:
This is how we're selecting which test(s) to run. The variable is set in the specific case defined later in the file.

Reply:
Could this be done without having to `cd` into each folder? Does sirun support specifying any path to a `meta.json`?

Reply:
> Could this be done without having to cd into each folder?

Sure, but you'll want the `$PWD` to be this directory anyway, so that the commands in the meta.json can use relative paths.

> Does sirun support specifying any path to a meta.json?

Not at the moment. I'm not sure how that's beneficial over just setting the working directory here though.